src/platforms/ctxlink/ctxLink_mode_led.c

```diff
@@ -0,0 +1,156 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+// file:	src\platforms\ctxLink\wdbp_mode_led.c
+//
+// summary:	Wdbp mode LED class, provides support for indicating the mode of ctxLink using a single LED
+////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include "general.h"
+#include "platform.h"
+#include "ctxLink_mode_led.h"
+
+mode_led_modes_e led_mode = mode_led_idle; // Initial state of the mode led
+mode_led_task_states_e mode_task_state = mode_led_idle_state;
+
+u_int32_t led_mode_timeout = 0;       // Used to time the on/off state of the led
```

Comment:
Unsure where this type is getting defined, but the C11 one is `uint32_t` please. `U` suffixes on the values being assigned would suppress a signed-unsigned conversion.

Reply:
Like you I don't know where this is defined, I assumed it was related to the WINC1500 driver source, but a search there does not reveal it.
Have searched my source and replaced with uint32_t.